projects/packages/plans/src/class-current-plan.php

```diff
@@ -324,11 +325,17 @@ public static function get_minimum_plan_for_feature( $feature ) {
 	 * @static
 	 *
 	 * @param string $feature The module or feature to check.
+	 * @param bool   $refresh_from_wpcom Refresh the local plan cache from wpcom.
 	 *
 	 * @return bool True if plan supports feature, false if not
 	 */
-	public static function supports( $feature ) {
+	public static function supports( $feature, $refresh_from_wpcom = false ) {
 		// Hijack the feature eligibility check on WordPress.com sites since they are gated differently.
```

Comment:
I think this comment needs to move down to right above the `$should_wpcom_gate_feature` variable.